src/pages/Form.tsx

```diff
@@ -0,0 +1,115 @@
+import { FunctionComponent } from "react";
+import formImage from "../images/ask_form.jpg";
+import logo from "../images/logo.svg";
+
+import {
+	Button,
+	VStack,
+	Flex,
+	Text,
+	Stack,
+	Image,
+	ButtonGroup,
+} from "@chakra-ui/react";
+
+import { GoMarkGithub } from "react-icons/go";
+import LoginFrom from "../components/loginForm";
+import RegisterForm from "../components/registerForm";
+
+interface SalFormProps {
+	currentForm: string;
```

Comment:
Make the page controls it's own state, there is no need to retrieve  the `currentPage` from parent as a prop.